src/components/chip/index.tsx

```diff
@@ -161,6 +165,7 @@ const Chip = ({
   resetSpacings,
   size = DEFAULT_SIZE,
   useSizeAsMinimum = true,
+  recorderTag,
```

Comment:
Should this have a default?

* Writing here but it's a general question for all of the "base" components (Chip, Icon, Image, Text, TouchableOpacity, View and TextField).

Reply:
We set our defaults in private.
In case our users use FullStory in their apps, we don't want to mess up with their rules. 
That's also the reason why we put the fsTagName prop before the props spreading :)